modules/extensions-support/pom.xml

```diff
@@ -51,11 +51,11 @@
             <artifactId>mule-core</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-spring-config</artifactId>
-            <version>${project.version}</version>
-        </dependency>
+        <!--<dependency>-->
+            <!--<groupId>org.mule.modules</groupId>-->
+            <!--<artifactId>mule-module-spring-config</artifactId>-->
+            <!--<version>${project.version}</version>-->
+        <!--</dependency>-->
```

Comment:
I commented it when I was trying some things, it turns out that is not required. I'll delete it